public/aboutus.html

```diff
@@ -0,0 +1,230 @@
+<p><img class="wysiwyg-text-align-center" src="/hc/en-us/article_attachments/202822865/Screen_Shot_2016-07-10_at_2.47.48_pm.png" alt=""></p>
+<p><strong>Privacy </strong></p>
+<p>The privacy of our members is important to us. We want you to feel safe sharing information online that you would be happy to share over the fence with a neighbour.</p>
+<ul>
+<li>We ask that all members use their real name when creating their account.</li>
+<li>We ask that all members verify their address</li>
+<li>We do not share your personal information with any advertisers or third parties</li>
+<li>SG Kampung is protected by a password and encrypted by HTTPS</li>
+<li>All data is stored in Singapore and is protected under Singaporen privacy laws</li>
+</ul>
+<p> </p>
+  </div>
+
+  <div class="article-attachments">
+    <ul class="attachments">
+
+    </ul>
+  </div>
+
+<input id="hdnTitle" type="hidden" value="SG Kampung Help Centre & FAQ" />
+
+</article>
+
+<aside class="article-sidebar side-column">
+  <div data-recent-articles></div>
+
+  <section class="related-articles">
+    <h3>Related articles</h3>
+    <ul>
+
+        <li>
+          <a href="/hc/en-us/articles/204179795-What-are-SG Kampung-Constituent-Leads-">What are SG Kampung Constituent Leads?</a>
+        </li>
+
+        <li>
+          <a href="/hc/en-us/articles/202376905-Getting-to-know-SG Kampung">Getting to know SG Kampung</a>
+        </li>
+
+        <li>
+          <a href="/hc/en-us/articles/201654899-Terms-of-Use">Terms of Use</a>
+        </li>
+
+        <li>
+          <a href="/hc/en-us/articles/202376695-Getting-started-">Getting started </a>
+        </li>
+
+        <li>
+          <a href="/hc/en-us/articles/203764279-Creating-the-right-account-">Creating the right account </a>
+        </li>
+
+    </ul>
+  </section>
+
+
+</aside>
+  </main>
+
+  <footer class="footer">
+  <div class="footer-inner">
+    <!-- Footer content -->
+    <h6>SG Kampung Community 2016</h6>
+    <a href="../public/aboutus.html" target="_blank">About Us</a>
+    <a href="../public/organizations.html">Organisations</a>
+    <a href="../public/events.html">Events</a>
+    <a href="../public/blog.html" target="_blank">Blog</a>
+    <a href="../public/leads.html" target="_blank">Leads</a>
+    <a href="../public/privacypolicy.html" target="_blank">Privacy Policy</a>
+    <a href="../public/tou.html" target="_blank">Terms of Use</a>
+    <a href="../public/help.html" target="_blank">Help</a>
+    <a class="last" href="../public/contactus.html" target="_blank">Contact Us</a>
+  </div>
+</footer>
+
+
+  <!-- / -->
+
+
+  <script type="text/javascript" src="//p4.zdassets.com/hc/en-us/locales-e6afffab8139470128341159c1ae50a017e43e46.js"></script>
+  <script src="https://SG Kampung.zendesk.com/auth/v2/host.js" data-brand-id="102805" data-return-to="https://help.SG Kampung/hc/en-us/articles/202364115-About-SG Kampung" data-theme="hc" data-locale="1" data-auth-origin="102805,true,true"></script>
```

Comment:
is a 404, many will have 404, subject to need to add functionality, these will eventually be deleted